test/hotspot/jtreg/gc/shenandoah/compiler/TestLinkToNativeRBP.java

```diff
@@ -23,36 +23,36 @@
 
 /**
  * @test
+ * @enablePreview
  * @bug 8259937
  * @summary guarantee(loc != NULL) failed: missing saved register with native invoke
  *
  * @requires vm.flavor == "server"
  * @requires ((os.arch == "amd64" | os.arch == "x86_64") & sun.arch.data.model == "64") | os.arch == "aarch64"
  * @requires vm.gc.Shenandoah
  *
- * @modules jdk.incubator.foreign
- *
  * @run main/othervm --enable-native-access=ALL-UNNAMED -XX:+UnlockDiagnosticVMOptions
  *                   -XX:+UseShenandoahGC -XX:ShenandoahGCHeuristics=aggressive TestLinkToNativeRBP
  *
  */
 
-import jdk.incubator.foreign.CLinker;
-import jdk.incubator.foreign.FunctionDescriptor;
-import jdk.incubator.foreign.SymbolLookup;
-import jdk.incubator.foreign.ValueLayout;
+import java.lang.foreign.CLinker;
+import java.lang.foreign.FunctionDescriptor;
 
 import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodType;
+
+import staticjava.lang.foreign.CLinker.C_INT;
 
 public class TestLinkToNativeRBP {
     static {
         System.loadLibrary("LinkToNativeRBP");
     }
 
-    final static CLinker abi = CLinker.systemCLinker();
-    static final SymbolLookup lookup = SymbolLookup.loaderLookup();
-    final static MethodHandle foo = abi.downcallHandle(lookup.lookup("foo").get(),
-            FunctionDescriptor.of(ValueLayout.JAVA_INT));
+    final static CLinker abi = CLinker.getInstance();
```

Comment:
This looks incorrect. This method was renamed to `systemCLinker`. Note that this test doesn't run as part of run-test-jdk_foreign.

Reply:
Yepp, fixing to 
```    
final static CLinker abi = CLinker.systemCLinker();
final static MethodHandle foo = abi.downcallHandle(TestLinkToNativeRBP.class.getClassLoader().findNative("foo").get(),
            FunctionDescriptor.of(ValueLayout.JAVA_INT));
```